configure.ac

```diff
@@ -797,6 +797,9 @@ fi
 
 AM_CONDITIONAL([HAVE_LIBNL], [test "$have_libnl" = "yes"])
 
+AC_CHECK_LIB([rdmacm], [rdma_lib_reset],
+		[AC_DEFINE([RDMA_LIB_RESET], [1], [rdma_lib_reset enabled])], [])
```

Comment:
Instead of RDMA_LIB_RESET please use DEFINED_RDMA_LIB_RESET. Please update the message as follows
```
[AC_DEFINE([DEFINED_RDMA_LIB_RESET], [1], [Define to 1 if rdma_lib_reset is defined])], [])
```